omop_file_validator.py

```diff
@@ -269,16 +311,52 @@ def check_csv_format(f, column_names):
     return results
 
 
-def run_checks(file_path, f):
-    file_name, file_extension = os.path.splitext(file_path)
-    file_path_parts = file_name.split(os.sep)
-    table_name = file_path_parts[-1]
-    print('Found CSV file %s' % file_path)
+def check_json_format(f, column_names):
+    results = []
+    idx = 1
+
+    try:
+        json_list = list(f)
+
+        for idx, json_str in enumerate(json_list, start=1):
+            json_obj = json.loads(json_str)
+            if len(json_obj.values()) != len(column_names):
+                column_mismatch_msg = 'Incorrect number of columns on line %s: %s' % (
+                    str(idx), json_str)
+                results.append([column_mismatch_msg, None, None])
+                break
+    except (json.JSONDecodeError, ValueError):
+        error_msg = f"The following exception was raised on line {idx}:\n\n{traceback.format_exc()}"
+
+        results.append([error_msg, None, None])
+
+        # print(
+        #     'Enclose all fields in double-quotes\n'
```

Comment:
I think you can remove this comment. Maybe print something like "Use json library to create the JSONL" if that's causing the errors.